### PR TITLE
Added Product Pie Chart To visualize top 10 product

### PR DESCRIPTION
**Pull request by harmlessprince**

### CODE REVIEW

File: src/views/Admin/ReportingTool.vue [hunk 1]
```diff
@@ -456,6 +519,30 @@ export default {
...
+      this.productPieData.datasets = [
+        {
+          barPercentage: 1,
+          barThickness: 12,
+          maxBarThickness: 16,
+          data: this.productPieData.dataSet,
+          backgroundColor: this.productPieData.bgColor,
+        },
+      ];
+      console.log(this.productPieData);
```
Comment (teebarg) [inline src/views/Admin/ReportingTool.vue:544]: lets remove log

File: src/views/Admin/ReportingTool.vue [hunk 2]
```diff
@@ -475,4 +562,9 @@ button {
   width: 100%;
   border-top: 2px solid rgba(75, 85, 99, 0.54);
 }
+.product-pie-card {
```
Comment (teebarg) [inline src/views/Admin/ReportingTool.vue:565]: Use tailwind class `flex items-center justify-center`

File: src/views/Admin/ReportingTool.vue [hunk 3]
```diff
@@ -340,7 +402,8 @@ export default {
           }
           return 0;
         });
-
+        this.noOfSalesMadeOnEachProduct =
+          this.reports.meta.noOfSalesMadeOnEachProduct;
         this.getSums(this.reports.meta.groupedDataByBranch);
       } catch (err) {
```
Comment (teebarg) [inline src/views/Admin/ReportingTool.vue:408]: Can we alert this error so we can know if anything goes wrong?

File: src/views/Admin/ReportingTool.vue (hunk 2 above)
Comment (harmlessprince) [reply src/views/Admin/ReportingTool.vue:565]: Tailwind is not been used in this project

**Review by teebarg — APPROVED** [review]

**Pull request merged**